src/python/ensembl/io/genomio/manifest/check_integrity.py

```diff
@@ -154,7 +154,7 @@ def _check_md5sum(self, file_path: Path, md5sum: str) -> None:
             if readable_hash != md5sum:
                 raise InvalidIntegrityError(f"Invalid md5 checksum for {file_path}")
 
-    def prepare_integrity_data(self) -> None:
+    def prepare_integrity_data(self) -> None:  # pylint: disable=too-many-branches
```

Comment:
:scream:

Reply:
Yeah, I'd rather have the fix available first then we can think about making pylint happy (or change the number of branches we allow after all)